code/MimeUploadValidator.php

```diff
@@ -88,11 +88,11 @@ public function getExpectedMimeTypes($tmpFile)
         // Get the mime types and their variations from mimevalidator
         $knownMimes = Config::inst()->get(get_class($this), 'MimeTypes');
         if (isset($knownMimes[$extension])) {
-            if (is_array($knownMimes[$extension])) {
```

Comment:
it always return with an array. if its a single or multiple its an array. i tested both single and multiple values.
thanks